tests/integration/test_camera_opencv.py

```diff
@@ -6,8 +6,7 @@
 
 import cv2
 
-from modules.camera import camera_factory
-
+from modules.camera import camera_factory, camera_config
```

Comment:
Please separate into 2 lines, and also keep 2 spaces between the imports and global constants.
```
from modules.camera import camera_factory
from modules.camera import camera_config


IMAGE_LOG_PREFIX = ...
```